contracts/ThalesRoyale/ThalesRoyale.sol

```diff
@@ -507,6 +507,14 @@ contract ThalesRoyale is Initializable, ProxyOwned, PausableUpgradeable, ProxyRe
         emit NewNumberOfRounds(_rounds);
     }
 
+    function fulfillOracleKeyHistoryData(bytes32 _oracleKey) public onlyOwner {
+        // season 1 -> season
+        for (uint i=1; i<=season; i++) {
```

Comment:
season should be a parameter

Reply:
We should make sure that we only set it if it was empty before

Reply:
done